upgrade/README.md

```diff
@@ -112,17 +112,23 @@ Upgrade sequence:
 19. If you enabled new Insights feature in step 8, please follow the post installation steps listed [here](https://docs.google.com/document/d/1Um_FvVip5GtTWdezN2wANz3QsIoGrlFS07lTxNk6Sw8/edit?tab=t.0#heading=h.odfvfs38x0e3)
 
     **Note**: If Pipeline Insights was enabled before upgrade, you need to edit the query for Currently Running Pipelines panel (inside Pipeline Insights dashboard, in the Grafana UI) and change metric name from `currently_executing_pipelines_new_ratio` to `currently_executing_pipelines_new` (do not forget to save your changes to the dashboard)
+
+    **Note**: In this version we have upgraded the grafana from 10.2.2 to 12.0.2 we can check in grafana URL.
```

Comment:
let's drop this info here..i tihnk it better fits in the release notes (both internal and external)... 
we just need to give headsup before upgrade so similar line to be added in step 9